packages/pg-v5/lib/fetcher.js

```diff
@@ -141,7 +141,7 @@ module.exports = heroku => {
 
   async function arbitraryAppDB(app) {
     // Since Postgres backups are tied to the app and not the add-on, but
-    // we require *an* add-on to interact with, make sure that that add-on
+    // we require *an* add-on to interact with, make sure that add-on
```

Comment:
`that that` is more often correct vs the average doubled word. But I don't think it's necessary here.